extensions/src/main/java/org/pentaho/platform/plugin/services/metadata/SessionCachingMetadataDomainRepository.java

```diff
@@ -372,6 +393,10 @@ protected String generateDomainIdCacheKeyForSession( IPentahoSession session ) {
     return DOMAIN_CACHE_KEY_PREDICATE + session.getId();
   }
 
```

Comment:
TODO i need to add javadoc comment or  #generateDomainIdCacheKeyForSession(

Reply:
see commit: https://github.com/pentaho/pentaho-platform/pull/5124/commits/e972c54c9fca2dee7942a398f53790d962cfed60